README.md

```diff
@@ -634,6 +634,7 @@ Thanks to all [contributors](https://github.com/thangchung/awesome-dotnet-core/g
 * [EquinoxProject](https://github.com/EduardoPires/EquinoxProject) - Full ASP.NET Core 2.0 application with DDD, CQRS and Event Sourcing.
 * [eShopOnContainers](https://github.com/dotnet/eShopOnContainers) - Microservices Architecture and Containers based Reference Application.
 * [guidance-identity-management-for-multitenant-apps](https://github.com/Azure-Samples/guidance-identity-management-for-multitenant-apps) - How to manage user identities in a multitenant app on Microsoft Azure, using Azure Active Directory for authentication.
+* [IdeaShare Angular-WebAPI](https://github.com/shakenovdev/Angular-ASP.NetCoreWebAPI-MySQL-CRUD-project) - Web App built on Angular 5 that interacts with WebAPI which has MySQL database. [Demo](http://ideashareapp.azurewebsites.net)
```

Comment:
Hi @shakenovdev, thank for your PR but we only accept repo that at least 3 month